src/tests/ftest/control/daos_admin_privileged.py

```diff
@@ -80,20 +81,20 @@ def test_daos_admin_format(self):
         # Prep server for format, run command under non-root user
         self.log.info("Performing SCM storage prepare")
         try:
-            storage_prepare(self.hostlist_servers, user, "dcpm")
+            server.storage_prepare(user, "dcpm")
         except ServerFailed as err:
             self.fail("Failed preparing SCM as non-root user: {}".format(err))
 
         # Prep server for format, run command under non-root user
         self.log.info("Performing NVMe storage prepare")
         try:
-            storage_prepare(self.hostlist_servers, user, "nvme")
+            server.storage_prepare(user, "nvme")
```

Comment:
@mjmac could you please advise whether this is still required? In any case, if not, let's remove it in a different PR.

Reply:
It's not required as long as the test harness is doing everything against a running daos_server, because daos_server now automatically runs nvme prepare on startup.

That having been said, the legacy `daos_server storage scan` command will fail without it because that code path does not read a config file and therefore can't auto-prepare based on the configured nr_hugepages value. As long as nothing depends on that, it should be safe to remove the explicit prepare.